src/owlwatch/README.md

```diff
@@ -20,6 +21,15 @@ A summary of the comparison including:
 
 **Order matters**, so comparisons have always a source schema and a target one. In order to be considered 'compatible', target schema needs to contain, at least, all properties in the source schema with same types. Thus, finding properties in the target schema that don't exist in the source one is not considered an error.
 
+### Check empty panels
+The output will show whether a certain panel(s) is emtpy or is ok for a paricular ES instance(s).
+```
```

Comment:
It would probably need some comments on how ES are discovered (if it supports only ES instances inside docker containers) and what is considered a `project`. Summarizing, write it for someone that knows nothing about Bitergia and just want to use this to check some panel she has in a ES instance in her laptop.